modules/host/negotiaterenewcontract.go

```diff
@@ -241,7 +241,7 @@ func (h *Host) managedVerifyRenewedContract(so *storageObligation, txnSet []type
 	}
 	// The WindowEnd for the new file contract must be further in the future
 	// than the WindowEnd for the existing file contract.
-	if fc.WindowEnd <= so.proofDeadline() {
+	if fc.WindowEnd < so.proofDeadline() {
```

Comment:
Uh oh.

This breaks compatibility with all v0.6.0 hosts. I see why you did it but you can't utilize it until everyone upgrades to 1.0, otherwise contract renewals won't work on our network.